src/Aspire.ProjectTemplates.9.0.net9/templates/aspire-empty/.template.config/template.json

```
@@ -0,0 +1,248 @@
+{
+  "$schema": "https://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Common",
+    ".NET Aspire",
+    "Cloud",
+    "Web",
+    "Web API",
+    "API",
+    "Service"
+  ],
+  "name": ".NET Aspire Empty App",
+  "defaultName": "AspireApp",
+  "description": "A project template for creating an empty .NET Aspire app.",
+  "shortName": "aspire",
+  "sourceName": "AspireApplication.1",
+  "preferNameDirectory": true,
+  "tags": {
+    "language": "C#",
+    "type": "solution",
+    "editorTreatAs": "solution"
+  },
+  "precedence": "9000",
+  "identity": "Aspire.Empty.CSharp.9.net9",
+  "thirdPartyNotices": "https://aka.ms/dotnet/aspire/8.0-third-party-notices",
```

```suggestion
  "thirdPartyNotices": "https://aka.ms/dotnet/aspire/9.0-third-party-notices",
```

---

Actually, given this just links to the file in the repo, let's just make it:
```suggestion
  "thirdPartyNotices": "https://aka.ms/dotnet/aspire/third-party-notices",
```

---

Wouldn't this become a problem when we get to 10.0, and the default link would point to the 10.0 branch?